tests/run-pass/floats.rs

```diff
@@ -12,4 +12,9 @@ fn main() {
     assert_eq!(5.0f32 as u32, 5);
     assert_eq!(5.0f32 as i32, 5);
     assert_eq!(-5.0f32 as i32, -5);
+
+    assert_eq!((1.0 as f32).max(-1.0), 1.0);
```

Comment:
some tests around weird floats like `NaN` and `inf` would be great

Reply:
I can copy-paste the libstd test suite if you want. I didn't because the implementation is just forwarding and we run the libstd tests [here](https://medium.com/@nettsundere/the-implementation-of-min-and-max-operations-for-floating-point-types-80d3616c084d).

Reply:
Oh right, nevermind